code/modules/mob/living/living.dm

```diff
@@ -752,6 +752,8 @@
 		var/altered_grab_state = pulledby.grab_state
 		if(resting && pulledby.grab_state < GRAB_KILL) //If resting, resisting out of a grab is equivalent to 1 grab state higher. wont make the grab state exceed the normal max, however
 			altered_grab_state++
+		else if(is_species(pulledby, /datum/species/squid) && pulledby.grab_state < GRAB_KILL) // If the puller is a squid, suction cups make it harder to break free
```

Comment:
I could probably merge this into a long IF statement, but prefer the readability